### ISSUE TITLE
Swift 2.3 Support

### ISSUE THREAD

**Issue by loganblevins** [IBAnimatable/IBAnimatable#269]
For those of us wishing to develop in Xcode 8 beta 4, but stick with Swift 2.3 rather than taking the large step of using Swift 3. I know you have a Swift 3 branch, but I only want to Swift 2.3.

**Comment by JakeLin** [comment]
@loganblevins thanks for your suggestion, we are focusing on swift3 at the moment. Could you like to help us with this? Once your PR is ready, we will create a `swift2.3` branch to maintain all the changes. Once Xcode 8 releases GM, we can merge into `master`.

**Comment by loganblevins** [comment]
@JakeLin Sure, I'll give it a shot.

**Comment by gkye** [comment]
Added a `swift 2.3` branch available here. https://github.com/IBAnimatable/IBAnimatable/tree/swift2.3
Description of the updates made can be found here  https://gist.github.com/gkye/50e8dabb531a50256549a2d21eab1080 @JakeLin @loganblevins check it out when you get a chance

**Comment by gkye** [comment]
@tbaranes

**Comment by JakeLin** [comment]
@gkye thanks 👍

**Comment by JakeLin** [comment]
I have updated README.md to let the user know about Swift 2.3 support.  @loganblevins can we close this issue now?

**Comment by loganblevins** [comment]
@JakeLin Sounds good! Thank you.

**Comment by JakeLin** [comment]
@loganblevins cool, please let me know if you want to help us with some tasks, for example, maintaining `swift2.3` branch, merging changes from master to swift2.3, improving the code quality like removing forced unwrapping code.

**Comment by loganblevins** [comment]
@JakeLin Sure, I could tie up some loose ends. Is there a group for contributors?

**Comment by JakeLin** [comment]
@loganblevins great, welcome onboard. We can discuss more details. Can you send me your email for Slack? You can find my email address on my Github page github.com/JakeLin, thanks.
